spec/requests/posts_request_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'rails_helper'
+
+RSpec.describe 'Posts', type: :request do
+  let(:user) { User.create(id: 1, Name: 'Tom', Photo: 'https://unsplash.com/photos/F_-0BxGuVvo', Bio: 'Teacher from Mexico.') }
+  let(:post) { Post.create(id: 1, author_id: user.id, Title: 'Post 3', Text: 'This is my third post') }
+
+  it 'should get index' do
+    get "/users#{user.id}/posts"
+    expect(response).to have_http_status(200)
+  end
+
+  it 'should get show' do
+    get "/users#{user.id}/posts#{post.id}"
+    expect(response).to have_http_status(200)
+  end
+
+  it 'returns correct show template' do
+    get "/users#{user.id}/posts#{post.id}"
+    expect(response).to render_template(:show)
+  end
```

Comment:
- [ ] Kindly fix your tests, all of them are failing, now I dont know if this is a configuration issue, but when I run them, I get the following result.
![image](https://user-images.githubusercontent.com/81932478/214147100-15dbed7a-3e80-46e9-abe0-a3b8f28d0459.png)

## Note that if it is a configuration issue, leave a comment for the next reviewer

Reply:
Hello @Munsa1, this feedback is not sufficient and provides no guidance on a way forward.
The tests are all successful on my side
![image](https://user-images.githubusercontent.com/11190489/214148801-546cea21-ad43-4061-9f20-def6ecd73c92.png)